dev/packages/alpha/cisco-0.0.2/dataset/asa/agent/stream/stream.yml

```diff
@@ -0,0 +1,15 @@
+{{#if input == "syslog"}}
```

Comment:
This line is not correct in the handlebars syntax and needs to split in multiple stream files.

Unfortunately this is a known issue, but it requires some advanced templating processing - we didn't introduce it.

Reply:
As https://github.com/elastic/package-registry/pull/346 was merged, we can start to use it in the packages but it will not be supported yet in the UI. I suggest we pick one of the two inputs for now.

@alakahakai Which one do you think is more important? syslog or file?

Reply:
syslog is more important, since file input is used for testing only. For handlebars, can we add the support for Ember Truth Helper addon?